src/async_impl/client.rs

```diff
@@ -1397,6 +1414,8 @@ impl ClientBuilder {
     /// Pass `None` to disable timeout.
     ///
     /// Default is 90 seconds.
+    ///
+    /// This only affects idle connections in the pool, not active connections.
```

Comment:
This (and the next one) feel less beneficial to the reader. I'd prefer to leave them off.